orttraining/tools/scripts/opset12_model_transform.py

```diff
@@ -25,9 +25,10 @@
 
 input_model_name = sys.argv[1]
 output_model_name = input_model_name[:-5] + '_opset12.onnx'
```

Comment:
might be nice to allow the user to specify the output file name too.
see argparse for a convenient way to handle command line args: https://docs.python.org/3/library/argparse.html